services/dbaas/backups.go

```diff
@@ -49,6 +49,26 @@ func SetPgSqlClusterBackupData(d *schema.ResourceData, clusterBackups *dbaas.Clu
 				backupEntry["cluster_id"] = *backup.Properties.ClusterId
 			}
 
```

Comment:
backup.properties nil check here. Rarely the backend can send this null for some strange reason

Reply:
@cristiGuranIonos solved in https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/325/commits/11214202ea81a5a470fc3683ba90e92dd1355bca